projects/sandbox/train/train.py

```diff
@@ -121,18 +124,23 @@ def main(
         num_ifos,
         sample_rate,
         fduration,
-        normalizer=standard_scaler,
+        scaler=standard_scaler,
     )
 
-    preprocessor.whitener.fit(background)
+    preprocessor.whitener.fit(kernel_length, *background)
```

Comment:
How has the behavior changed now?

Reply:
Behavior hasn't changed, just the `ml4gw` API